app/src/main/java/org/thoughtcrime/securesms/trustedIntroductions/jobs/TrustedIntroductionSendJob.java

```diff
@@ -100,12 +128,52 @@ private TrustedIntroductionSendJob(@NonNull RecipientId introductionRecipientId,
 
 
```

Comment:
I get the function usage, but maybe a small description would be beneficial to understand better